docs/admin/runai-setup/maintenance/alert-monitoring.md

```diff
@@ -53,93 +41,82 @@ This documentation outlines the steps required to set up Alertmanager within the
            alertmanagerConfigSelector:
               matchLabels:
                  alertmanagerConfig: runai
-        EOF
 
-3. Exposing the Alertmanager Service
+2. Validate that your alertmanager instance has started:
 
-        cat <<EOF | kubectl apply -f - 
-        apiVersion: v1
-        kind: Service
-        metadata:
-           name: alertmanager-runai
-           namespace: runai
-        spec:
-           type: NodePort
-           ports:
-           - name: web
-              nodePort: 30903
-              port: 9093
-              protocol: TCP
-              targetPort: web
-           selector:
-              alertmanager: runai
-        EOF
+        kubectl get alertmanager -n runai
+
+3. Validate that prometheus operator has created a service for alertmanager:
+
+         kubectl get svc alertmanager-operated -n runai
 
 ## Configuring Prometheus to Send Alerts
 
 1. Edit the Prometheus configuration:
 
       `kubectl edit prometheus runai -n runai`
 
-2. Add the following to the `alerting` section:
+2. Add the following to the `spec.alerting` section:
 
         alerting:
            alertmanagers:
               - namespace: runai
-                name: alertmanager-runai
+                name: alertmanager-operated
                 port: web
 
 3. Save and exit the editor. The configuration will be automatically reloaded.
 
 ## Configuring Alertmanager for Custom Email Alerts
+In each step copy the content to a file and apply it to the cluster using `kubectl apply -f`
 
 1. Add your smtp password as a secret:
 
-        cat <<EOF | kubectl apply -f 
         apiVersion: v1
         kind: Secret
         metadata:
-           name: smtp-password
+           name: alertmanager-smtp-password
            namespace: runai
         stringData:
            password: "your_smtp_password"
-        EOF
-
-2. Edit the Alertmanager configuration:
 
-       `kubectl edit alertmanagerconfig -n runai`
-
-3. Add to the `spec` section, a new receiver configuration to send alerts via email:
-
-        receivers:
-        - name: 'email'
-           emailConfigs:
-           - to: 'your_email@example.com'
-              from: 'alertmanager@example.com'
-              smarthost: 'smtp.yourmailprovider.com:587'
-              authUsername: 'your_username'
+2. Apply the `alertmanagerconfig`, replace the relevant smtp details with your own:
```

Comment:
```suggestion
2. Apply the `alertmanagerconfig`, replace the relevant smtp details with your own:
```
How do you apply? Is there a command for this?

Reply:
added a description